packages/plugins/ganache/src/index.js

```diff
@@ -53,8 +53,6 @@ class Ganache {
     // => https://github.com/trufflesuite/ganache-cli/issues/558
     this.embark.fs.ensureDirSync(blockchainConfig.datadir);
 
-    const hasAccounts = blockchainConfig.accounts && blockchainConfig.accounts.length;
```

Comment:
Why do we no longer need to check for this condition?

Reply:
Back in the day, I added that condition so that we didn't generate account when using custom accounts (`hasAccounts`), but I discovered yesterday that Ganache always creates accounts as "node accounts", and the problem is, those accounts changes from one run to the other.
That means that if you run one time, you init some stuff with one account, and then, in an other run, you get a new account, so it fails.
So I just get rid of that condition, since it didn't work as I thought and it doesn't make sense to have it in hindsight.